Makefile

```diff
@@ -91,45 +102,26 @@ else
 	gocov-html < cover.json > cover/index.html
 endif
 
-# Build chaos-daemon binary
-chaosdaemon:
-	$(CGOENV) go build -ldflags '$(LDFLAGS)' -o bin/chaos-daemon ./cmd/chaos-daemon/main.go
-
-bin/pause: ./hack/pause.c
-	cc ./hack/pause.c -o bin/pause
-
-# Build manager binary
-manager:
-	$(GO) build -ldflags '$(LDFLAGS)' -o bin/chaos-controller-manager ./cmd/controller-manager/*.go
-
-chaosfs:
-	$(GO) build -ldflags '$(LDFLAGS)' -o bin/chaosfs ./cmd/chaosfs/*.go
-
-chaos-dashboard:
-ifeq ($(SWAGGER),1)
-	make swagger_spec
-endif
-ifeq ($(UI),1)
-	make ui
-	hack/embed_ui_assets.sh
-endif
-	$(CGO) build -ldflags "$(LDFLAGS)" -tags "${BUILD_TAGS}" -o bin/chaos-dashboard cmd/chaos-dashboard/*.go
```

Comment:
seems that we can not build binary later

Reply:
`make binary` has been added :smile_cat: